scripts/Install-EndpointAgentTcpNetworkTests.ps1

```diff
@@ -239,7 +239,7 @@ process {
     }
 
     # Always enable TCP Network Tests feature.
```

Comment:
Nit: update this comment too?

Reply:
Thanks!  Sorry to be a further pain, but do you think we should rename this file now?